### DISCUSSION TITLE
说明

### DISCUSSION THREAD

**Discussion by lhlyu** [icon-space/IconSpace#1] [Announcements]
原项目[iconpark](https://github.com/bytedance/iconpark)由于长时间未维护及更新，本项目由此诞生。
现在急需有志之士参与项目：
1. 本人不会画图，希望有人可以根据用户的提案，绘制SVG图标，建议以原项目[设计](https://bytedance.feishu.cn/docs/doccnU0WZ2HEsYBINa9TfgDtAzP)为标准。
2. 由于[原项目网站](https://iconpark.oceanengine.com/official)未开源，希望有人可以开发一个网站。
3. 希望有人可以设计一个组织的logo。

**Comment by AH-dark** [comment]
![image](https://user-images.githubusercontent.com/51253685/221911565-eb060abe-d084-41b4-8cba-044111198e7f.png)

I got an invitation to an organization from you, but I don't even know who you are or what this organization does. It seems impolite to send an invitation without notice or consultation.

**Reply by lhlyu** [reply]
正如我上面说的[iconpark](https://github.com/bytedance/iconpark)项目长期没有维护，这个项目是它的复刻，想另起炉灶，我关注的人很少，我只是把我关注人里最近活跃的邀请了一番，如果对你造成了影响见谅。
